### ISSUE TITLE
Unable to run the app after adding the package.

### ISSUE THREAD

**Issue by hansyulian** [JuanSeBestia/react-native-wifi-reborn#54]
I'm using react native 0.61.5, for some reason i can't use `react-native run-android` after installing this package. here is the error reported

```
> Task :app:processSandboxDebugManifest FAILED

See http://g.co/androidstudio/manifest-merger for more information about the manifest merger.


Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.5/userguide/command_line_interface.html#sec:command_line_warnings
213 actionable tasks: 2 executed, 211 up-to-date
D:\_crypto-defense-source-codes\Zynesis\nzia-wallet\android\app\src\debug\AndroidManifest.xml:13:7-34 Error:        
        Attribute application@allowBackup value=(false) from AndroidManifest.xml:13:7-34
        is also present at [com.thanosfisherman.wifiutils:wifiutils:1.5.0] AndroidManifest.xml:18:9-35 value=(true).
        Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:7:5-117 to override.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processSandboxDebugManifest'.
> Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml:13:7-34
        is also present at [com.thanosfisherman.wifiutils:wifiutils:1.5.0] AndroidManifest.xml:18:9-35 value=(true).
        Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:7:5-117 to override.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 16s

error Failed to install the app. Make sure you have the Android development environment set up: https://facebook.github.io/react-native/docs/getting-started.html#android-development-environment. Run CLI with --verbose flag for more details.
Error: Command failed: gradlew.bat app:installSandboxDebug -PreactNativeDevServerPort=8081
D:\_crypto-defense-source-codes\Zynesis\nzia-wallet\android\app\src\debug\AndroidManifest.xml:13:7-34 Error:
        Attribute application@allowBackup value=(false) from AndroidManifest.xml:13:7-34
        is also present at [com.thanosfisherman.wifiutils:wifiutils:1.5.0] AndroidManifest.xml:18:9-35 value=(true).
        Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:7:5-117 to override.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processSandboxDebugManifest'.
> Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml:13:7-34
        is also present at [com.thanosfisherman.wifiutils:wifiutils:1.5.0] AndroidManifest.xml:18:9-35 value=(true).
        Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:7:5-117 to override.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 16s
```

**Comment by eliaslecomte** [comment]
That is stupid. I will make sure that this allowBackup is removed from the WifiUtils library that we are using in a future version.

You can now fix it yourself by making sure you have tools:replace="android:allowBackup" in your <application> tag (android/app/src/main/AndroidManifest file)

Example:
`<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    package="yourpackage">

    <application
        android:name=".MainApplication"
        android:allowBackup="false"
        tools:replace="android:allowBackup">`

**Comment by eliaslecomte** [comment]
See https://github.com/ThanosFisherman/WifiUtils/pull/57

**Comment by Rapsssito** [comment]
Fixed with #53!
